lib/StripesFormWrapper.js

```diff
@@ -2,9 +2,9 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router';
 import { submit } from 'redux-form';
+import { intlShape } from 'react-intl';
 import { LastVisitedContext } from '@folio/stripes-core/src/components/LastVisited';
-
-import StripesFormModal from './StripesFormModal';
+import { ConfirmationModal, injectIntl } from '@folio/stripes-components';
```

Comment:
@cherewaty, I'm noticing `injectIntl` is not exported by stripes-components yet.

>WARNING in ./node_modules/@folio/stripes-form/lib/StripesFormWrapper.js 144:26-36
"export 'injectIntl' was not found in '@folio/stripes-components'

Reply:
https://github.com/folio-org/stripes-form/pull/34